packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts

```diff
@@ -150,7 +150,9 @@ export default class JSONSchemaBridge extends Bridge {
         fieldInvariant(name, !!definition);
       } else if (definition.type === 'object') {
         fieldInvariant(name, !!definition.properties);
-        definition = definition.properties[next];
+        definition =
+          definition.properties[next] ||
+          definition.properties[next.slice(2, -2)];
```

Comment:
How about making a function that'd trim the `[""]`? Then we'd have a similar one for the code in lines 301-303.